xcvm/lib/core/src/lib.rs

```diff
@@ -72,7 +73,7 @@ where
 	}
 
 	#[inline]
-	pub fn call<T>(self, protocol: T) -> Result<Self, T::Error>
+	pub fn call_raw_protocol<T>(self, protocol: T) -> Result<Self, T::Error>
```

Comment:
Why renaming this???

Reply:
because there is call_raw. and this is also call_RAW_protocol. 

likely we will have call_lazy and call_LAZY_protocol. so it will be just call, no lazy word. 

so to avoid confusion to split raw and lazy non raw calls. right?

Reply:
so Mars will do Mars protocol with only ToRaw<Network> trait. Later the may add ToRaw<Polygone>. 
And later ToXcvm trait.

so call_raw_protocol requires ToRaw,

while call_procol requires ToXcvm

Reply:
ok I revert back, but will request the change to have consistent naming.